src/content/posts/2025-pyladiescon-is-back.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: "PyLadiesCon is back!"
+meta_title: "PyLadiesCon is back!"
+description: "PyLadiesCon team had a great time at PyCon US 2025 in Pittsburgh, PA"
+date: 2025-06-23T23:00:00Z
+image: "/images/posts/pyladiescon-is-back.png"
+categories: ["Blog Post",]
+authors: ["Georgi"]
```

Comment:
Your author information doesn't show up in the blog post. Maybe you need to add yourself to the src/content/authors. eg https://github.com/pyladies/global-conference/tree/main/src/content/authors

Reply:
Oh yeah I didn’t create one for myself 😅. I’ll do that tomorrow.

Reply:
@Mariatta updated.

Reply:
Looks great now, thanks!